pkgs/applications/networking/feedreaders/newsboat/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchzip, asciidoc, sqlite, curl, pkgconfig, libxml2, stfl
+, json-c-0-11 , ncurses , gettext, libiconv, makeWrapper, perl, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  name = "newsboat-${version}";
+  version = "2.10.1";
+
+  src = fetchzip {
+    url = "https://newsboat.org/releases/${version}/${name}.tar.xz";
+    sha256 = "019bm8j9vbpj39vs6xhrj34cd9ipjyqpwkl5psaks2w6g7wzyp1p";
+  };
+
+  buildInputs
+    # use gettext instead of libintlOrEmpty so we have access to the msgfmt
+    # command
+    = [ asciidoc sqlite curl pkgconfig libxml2
+      stfl json-c-0-11 ncurses gettext perl libiconv ]
+      ++ stdenv.lib.optional stdenv.isDarwin makeWrapper;
+
+  preBuild = ''
+    substituteInPlace config.sh \
+      --replace "ncurses5.4" "ncurses"
+    substituteInPlace ./Makefile \
+      --replace "a2x" "${asciidoc}/bin/a2x --no-xmllint"
+  '';
+
+  installFlags = [ "prefix=$(out)" ];
+
+  postInstall = stdenv.lib.optionalString stdenv.isDarwin ''
+    cp -r $src/contrib $out
+    for prog in $out/bin/*; do
+      wrapProgram "$prog" --prefix DYLD_LIBRARY_PATH : "${stfl}/lib"
+    done
+  '';
+
+  meta = {
```

Comment:
most of the derivations use `meta = with stdenv.lib; {` which makes the expression less verbose as you don't have to type `stdenv.lib.{any-value}` all the time.